### PR TITLE
Update docs for event changes

### PR DESCRIPTION
**Pull request by Ernst79**
Update docs for event changes in https://github.com/Bluetooth-Devices/bthome-ble/pull/18

### CODE REVIEW

File: bthome_v1.html [hunk 1]
```diff
@@ -1175,11 +1150,12 @@ <h4>Events</h4>
...
                 </td>
-                <td>turn_off</td>
+                <td>single_press</td>
+                <td></td>
                 <td>
-                  <code>023B02</code>
+                  <code>023A02</code>
                 </td>
-                <td>Switch event "turn_off"</td>
+                <td>single_press</td>
```
Comment (balloob) [inline bthome_v1.html]: why do we have `press` and `single_press`

File: index.html [hunk 2]
```diff
@@ -403,6 +402,13 @@ <h3 id="bthome-data-format">BTHome Data format</h3>
             with a factor 0.01, this becomes a humidity of 50.55%.
           </li>
         </ul>
+        <p>
+          It is strongly advised to use object ids in numerical order (from low to high) in your advertisement.
```
Comment (balloob) [inline index.html]: Instead of strongly advising it, why not make it mandatory?

File: index.html [hunk 3]
```diff
@@ -1344,14 +1334,19 @@ <h3 id="encryption">Encryption</h3>
...
+        <h3 id="multiple_measurements">Multiple measurements/events of the same type</h3>
         <p>
           If you want to send multiple measurements of the same type, e.g. three temperatures,
           you can just add multiple measurements of the same type to the payload. A
           postfix will be added to the measurement name (e.g. temperature_2) in the order
           of which you define the measurements. Note that this implies that you will need to use
           the same order in each advertisement, to prevent measurements being assigned to the wrong
           entity. If only one measurement of a certain type is sent, no postfix will be used.
+          <br /><br />
+          For events, a special event `0x00` (None) is available, e.g. in case you have a
```
Comment (balloob) [inline index.html]: This is now twice in this doc. I think that we can just make this section about just Multiple measurements, and have a multiple events next to the events table.

File: bthome_v1.html (hunk 1 above)
Comment (Ernst79) [reply bthome_v1.html]: Yeah, that does not really make a lot sense. Removed `single_press`. 
Also `long_single_press` is removed, as we have `long_press`

File: index.html (hunk 2 above)
Comment (Ernst79) [reply index.html]: Agreed, changed it.

File: index.html (hunk 3 above)
Comment (Ernst79) [reply index.html]: Yeah, I was struggling a bit with this, as the Multiple measurement section was at the end of the page, so after the event section, where I wanted to explain this. I now moved the "multiple measurement" explanation to below the table with all the sensor types, and moved the part about events to below the event types table.

**Comment by Ernst79** [comment]
@balloob Thanks for taking the time to look into all my changes. Much appreciated!

**Pull request merged**